backend/ops_api/tests/ops/workflows/test_workflow_procurement.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+from models import Package, PackageSnapshot, WorkflowAction, WorkflowInstance, WorkflowStepInstance, WorkflowStepStatus
+from models.workflows import WorkflowTriggerType
+
+
+@pytest.mark.usefixtures("app_ctx", "loaded_db")
+def test_creating_procurement_tracker_workflow(loaded_db):
+    # Create a mock WorkflowStepInstance with the required attributes
+
+    # Where and how should we create a Procurement workflow?
+    # Do we want to rework the dependencies
+
+    procurement_workflow = Mock(spec=WorkflowInstance)
+    procurement_workflow.workflow_template_id = 2
+    procurement_workflow.associated_id = 1
+    procurement_workflow.associated_type = WorkflowTriggerType.AGREEMENT
+    procurement_workflow.workflow_action = WorkflowAction.GENERIC
+
+    # workflow_template = Mock(spec=WorkflowTemplate)
```

Comment:
should this be commented out?

Reply:
I forgot that other test file was still in there.  That whole test file is going to be redone as we start working on the workflow side of this.